.github/labeler.yml

```diff
@@ -393,8 +393,8 @@
 "ABI break":
   - changed-files:
     - any-glob-to-any-file:
-      - 'TSRM/*.h'
-      - 'Zend/*.h'
+      - 'TSRM/TSRM.h'
```

Comment:
Can we have these lists generated by a small php script which can be added into https://github.com/php/php-src/blob/master/.github/actions/verify-generated-files/action.yml to assert all lists are always in sync and up-to-date?

Reply:
See https://github.com/php/php-src/pull/15682#issuecomment-2323058771; Windows only; I wouldn't know how to do this for other systems (i.e. via autoconf).

Reply:
As this PR proposes to hardcode the files, we do not need to generate them on each local build, but we need to assert that the hardcoded lists are up-to-date - for that purpose, there is https://github.com/php/php-src/blob/master/.github/actions/verify-generated-files/action.yml CI and we need a simple PHP script which will discover all *.h files.

In that script, published files can be hardcoded and the rest assumed as non-published. The script can then locate the lists in all files where the hardcoded lists are to be present and replace the lists will up-to-date lists.

The CI job would then check if the files in git are matching the generated outputs the same as we do for stubs and other generated files.

Reply:
This PR does not propose the list; that already happened via #15682. I agree that auto-generating and checking the list would be good, but I still have not been able to figure out how to do that on non Windows systems; and doing it only on Windows *might* miss headers from Windows incompatible stuff.